sections/popular-products.liquid

```diff
@@ -0,0 +1,145 @@
+{{ 'popular-products.css' | asset_url | stylesheet_tag }}
+<h3 class="title title--popular-products">{{ section.settings.title }}</h3>
+<div class="popular-products__container">
+  <div class="product-block">
+    {% for block in section.blocks %}
+      <div class="product-block__container">
+        <div class="product-block__img-container">
+          <img
+            src="{{ block.settings.product.images[0] | image_url: width: 145, height: 115 }}"
+            alt="{{ block.settings.product.title }}"
+            class="product-block__img">
+          {% if block.settings.product.metafields.custom.informative_item %}
+            <p class="product-block__metadata {% if block.settings.product.metafields.custom.informative_item == 'solo web' %}product-block__metadata--green{% endif %}">
+              {{ block.settings.product.metafields.custom.informative_item }}
+            </p>
+          {% endif %}
+        </div>
+        <a
+          href="{{ block.settings.product.url }}"
+          class="product-block__name"
+          aria-label="Ver producto {{ block.settings.product.title }}">{{ block.settings.product.title }}</a>
+        <div class="product-block__detail-container">
+          <div class="product-block__detal-price-container">
+            <p class="product-block__price">{{ block.settings.product.price | money }}</p>
+            <p class="product-block__price-comparation">{{ block.settings.product.compare_at_price | money }}</p>
```

Comment:
Sorry, I didn't see this before. You should use a conditional so an empty block is not rendered
```suggestion
            {% if block.settings.product.compare_at_price > block.settings.product.price %}<p class="product-block__price-comparation">{{ block.settings.product.compare_at_price | money }}</p>{% endif %}
```